linera-chain/src/chain.rs

```diff
@@ -46,6 +49,15 @@ pub static NUM_BLOCKS_EXECUTED: Lazy<IntCounterVec> = Lazy::new(|| {
     )
     .expect("Counter can be created")
 });
+pub static BLOCK_EXECUTION_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "block_execution_latency",
+        "Block execution latency",
+        // Can add labels here
+        &[]
```

Comment:
This is where I mentioned we can add labels like `chain_id` or whatever else we want